src/resolve/resolved-resolv-conf.c

```diff
@@ -323,6 +323,25 @@ static int write_stub_resolv_conf_contents(FILE *f, OrderedSet *dns, OrderedSet
         return fflush_and_check(f);
 }
 
+void manager_symlink_stub_to_uplink_resolv_conf(void) {
+        _cleanup_free_ char *fname = NULL;
+        int r;
+        if (access(PRIVATE_UPLINK_RESOLV_CONF, R_OK) != 0) {
+                log_debug("Uplink %s is missing.", PRIVATE_UPLINK_RESOLV_CONF);
```

Comment:
if you have an error number, pass it to the logging calls, even if you don't use %m. We use structured logging after all, and the error number will show in in the structured fields in that case.

```
log_debug_errno(errno, …
```

Reply:
we try to include in our log messages a hint about the consequence of the failure. maybe hre: "Uplink %s file is missing, not symlinking it to %s.".